package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless

```diff
@@ -194,13 +204,11 @@ wireless.foreach_radio(uci, function(radio, index, config)
 		return
 	end
 
-	local channel = get_channel(radio, config)
 	local htmode = get_htmode(radio)
 	local beacon_interval = config.beacon_interval()
 
 	uci:delete('wireless', radio_name, 'disabled')
 
-	uci:set('wireless', radio_name, 'channel', channel)
```

Comment:
We're now missing channel configuration for `band == '2g'`. I think we can just call `set_channels()` before the band-specific part, and add a band check inside `set_channels`, so the outdoor_chanlist is only applied for 5g.

Reply:
Whoops, good catch!